src/index.coffee

```diff
@@ -26,6 +26,13 @@
 #      default and reverse models.
 #   HUBOT_MARKOV_RESPOND_CHANCE - The probability, between 0.0 and 1.0, that Hubot will respond
 #      un-prompted to a message by using the last word in the message as the seed. Default: 0.
+#   HUBOT_MARKOV_IGNORE_MESSAGE_LIST - Comma-separated list of sub-phrases to ignore from
+#      default and reverse models. Is a substring match for every message
+#   HUBOT_MARKOV_LEARNING_LISTEN_MODE - 'catch-all' (default): learn message if no other scripts
+#      process message. 'hear-all': learn every message, regardless if another script handled it
```

Comment:
From the code, it looks like if this or the `RESPOND_LISTEN_MODE` setting below are any values other than `catch-all` or `hear-all`, they'll be used as an exact pattern that can be used to explicitly trigger the relevant behavior. That's neat! Can we document that here so users can find out about it?